sequencing/gen_ancestry/dxapp.json

```diff
@@ -1,35 +1,38 @@
 {
   "name": "gen_ancestry",
   "title": "Determine Ancestry from PCA",
-  "summary": "gen_ancestry",
-  "dxapi": "1.0.0",
-  "version": "0.0.1",
-  "categories": [
+  "summary": "Determines ancestry from PCA eigenvector file using custom R script.",
+  "tags": [
     "Statistics"
   ],
+  "dxapi": "1.0.0",
+  "version": "1.0.0",
   "inputSpec": [
     {
       "name": "evec_fn",
```

Comment:
Maybe change the input parameter
"evec_fn" -> "pca_evec" to follow the input/output nomenclature?
Also will need to update the variable used on the app script.